packages/react/src/button/Button.css.ts

```diff
@@ -121,7 +129,6 @@ export const button = recipe({
             backgroundColor: subtleAccentColorVar,
           },
           '&[aria-disabled="true"]': {
```

Comment:
We will use data attributes in styling,
`&[data-disabled="true"]`

Reply:
I think we should keep it as it is. We need aria-disabled for keyboard interactions.

Reply:
we'll keep both aria-disabled and data-disabled on the element - but only use `data-*` when styling - since that is the contract - whereas `aria-*` attributes are just a side effect of the intent

Reply:
understood bhai!